pom.xml

```diff
@@ -19,6 +19,7 @@
   <properties>
     <version.junit>4.12</version.junit>
     <version.assertj>3.10.0</version.assertj>
+    <version.system.rules>1.18.0</version.system.rules>
```

Comment:
No objections about it, however I removed system-rules in https://github.com/fabric8-launcher/launcher-booster-catalog-service/commit/188379e600f0d5aa5afd02fc27965a14c58bcd65#diff-7284a18723a53882311a47386760e680 because for some reason the system properties were not reset when the test was over and by reading the docs they suggested using System.setProperty directly.

Reply:
Oh that's interesting. Do you happen to know if there was an issue open?

Also, notice that I'm using EnvVars instead. I can redo that though, no worries.